bubble_sort.py

```diff
@@ -28,9 +28,18 @@ def bubble_sort(lista_numeros: list[int]):
 
     >>> bubble_sort([1, 3, 1])
     [1, 1, 3]
-    """
+    """  
+    def bubble_sort(lista_numeros): 
+        n = len(lista_numeros) 
+        for i in range(n): 
+            Troca = False 
+            for j in range(n-i-1): 
+                if lista_numeros[j] > lista_numeros[j+1] : 
+                    lista_numeros[j], lista_numeros[j+1] = lista_numeros[j+1], lista_numeros[j]
+                    Troca = True
+            if Troca == False: 
+                break
+        return lista_numeros
```

Comment:
```suggestion
    n = len(lista_numeros) 
    for i in range(n): 
        Troca = False 
        for j in range(n-i-1): 
            if lista_numeros[j] > lista_numeros[j+1] : 
                lista_numeros[j], lista_numeros[j+1] = lista_numeros[j+1], lista_numeros[j]
                Troca = True
        if Troca == False: 
            break
    return lista_numeros
```
Na realidade, tem uma função definida dentro de outra, o que já começa com problema.